package.json

```diff
@@ -26,6 +26,7 @@
     "@types/node": "^20.11.30",
     "@types/nunjucks": "^3.2.6",
     "eslint": "^8.57.0",
+    "globals": "^15.8.0",
```

Comment:
what is this?

Reply:
Oh I see. Okay got rid of this!

Reply:
@erinmcj opening this one again, would expect to see a file change for eslint version number

Reply:
Oh so the eslint on line 28 should be 9.0.0?

Reply:
Tried going to eslint version 9 and it caused problems because typescript eslint doesn't support it yet https://typescript-eslint.io/users/dependency-versions/